boards/f4vi1/Makefile.features

```diff
@@ -1,6 +1,7 @@
 # Put defined MCU peripherals here (in alphabetical order)
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_gpio
```

Comment:
We could also move `periph_gpio` feature entries from all stm or even cortexm based boards to the CPU files, as these are not dependent on any board specific configuration and they are implemented for all these CPUs

Reply:
My understanding was that the feature being available also depends on the board's configuration in ```periph_conf.h``` and thus might override a CPU definition.